src/state/claimables/airdropsStore.ts

```diff
@@ -222,13 +223,13 @@ export const useAirdropsStore = createQueryStore<AirdropsQueryData, AirdropsPara
           if (result) newCacheEntry = { ...cacheEntry, data: result };
         }
 
-        if (!newAirdrops && !newCacheEntry) return state;
-
         return {
-          ...state,
           airdrops: newAirdrops,
```

Comment:
is it possible for `newAirdrops` to remain unset from line 214 and this to overwrite the airdrops state with null?

Reply:
Possible but shouldn't be an issue. If there are no claimables in `airdrops` when this function runs, the paginated data almost certainly wasn't being used. So it being set to `null` is fine since `getData` should have the data that matters.